UDV-Core/src/Utils/BaseDemo/js/BaseDemo.js

```diff
@@ -408,49 +388,101 @@ export class BaseDemo {
             source: wmsElevationSource,
         });
         this.view.addLayer(wmsElevationLayer);
+    }
 
-        // ********* ADD 3D BUILDING LAYER (3D Tiles)
-        // This building layer represents Lyon in 2015 and is served from
-        // grand lyon alpha server
+    /**
+     * Adds a 3D Tiles layer to the iTowns 3D view.
+     * @param {string} layer The name of the type of object to add to the view. This name should
+     * be one of the properties of the 3DTilesLayer object (in .json config file).
+     */
+    add3DTilesLayer(layer) {
+        //  ADD 3D Tiles Layer
         let $3dTilesLayer = new itowns.GeometryLayer(
-            this.config['3DTilesLayerID'], new THREE.Group());
-        $3dTilesLayer.name = 'Lyon-2015';
+            this.config['3DTilesLayer'][layer]['id'], new THREE.Group());
+        $3dTilesLayer.name = 'Lyon-2015-'.concat(layer);
         $3dTilesLayer.url =
-            this.config['3DTilesLayerURL'];
+            this.config['3DTilesLayer'][layer]['url'];
         $3dTilesLayer.protocol = '3d-tiles';
-        $3dTilesLayer.overrideMaterials = true;
+        let material;
+        if (layer === 'building') {
+            material = new THREE.MeshLambertMaterial({ color: 0xFFFFFF });
+        } else if (layer === 'relief') {
+            material = new THREE.MeshLambertMaterial({ color: 0xD2B48C });
+        } else if (layer === 'water') {
+            material = new THREE.MeshLambertMaterial({ color: 0xB0E0E6 });
+        }
```

Comment:
These colors should be optional parameters defined in the generalDemoConfig.json file.